### PR TITLE
Rest client

### PR DESCRIPTION
**Pull request by Saadnajmi**
Added a rest client that can GET events, and PUT/POST/DELETE events.

### CODE REVIEW

File: mobile/Common/Common.Common/Models/Event.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        public string StartTime { get; set; }
+
+        //TODO: Make this a DateTime Object that's compatible cross-platform
+        public string EndTime { get; set; }
+
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        public string Location { get; set; }
```
Comment (zuhairp) [inline mobile/Common/Common.Common/Models/Event.cs]: Heads up, this'll be a Location object

Comment (Saadnajmi) [inline mobile/Common/Common.Common/Models/Event.cs]: How is that gonna be easily deserialized?

Comment (zuhairp) [inline mobile/Common/Common.Common/Models/Event.cs]: JSON.net is very powerful, it'll figure it out

Comment (zuhairp) [inline mobile/Common/Common.Common/Models/Event.cs]: Set it up in such a way that if a field doesn't show up, it just sets it to null (that should be the default)

**Pull request closed without merging**